jsdoc/tag-defs/access.js

```diff
@@ -1,13 +1,16 @@
-module.exports = function(accessTagTransform, trimWhitespaceTransform) {
-  var name = 'access'
-  accessTagTransform.addTag(name);
-  accessTagTransform.addValue('private');
-  accessTagTransform.addValue('protected');
-  accessTagTransform.addValue('public');
-
+module.exports = function(createDocMessage) {
+  STANDARD_VALUES = ['private', 'protected', 'public'];
```

Comment:
Without the `var`, this is an implicit global variable, was this the intention by using all caps snake case?